roles/repositories/README.md

```diff
@@ -0,0 +1,108 @@
+theforeman.foreman.repositories
```

Comment:
I wonder if `theforeman.foreman.content` is not a more appropriate name? but wouldn't block on this

Reply:
Content seems so broad spectrum that given this is focused on repositories, I would stick to that personally.

Reply:
wfm, was just a thought :)